packages/core/src/app/shipping/MultiShippingFormV2.test.tsx

```diff
@@ -79,6 +79,38 @@ describe('MultiShippingFormV2 Component', () => {
         });
     });
 
+    it('shows error when creating a new destination if destination 1 is incomplete.', async () => {
+        jest.spyOn(checkoutState.data, 'getConsignments').mockReturnValue([]);
+        jest.spyOn(checkoutState.data, 'getCheckout').mockReturnValue({
+            ...getCheckout(),
+            consignments: [],
+        });
+
+        render(
+            <CheckoutProvider checkoutService={checkoutService}>
+                <LocaleContext.Provider value={localeContext}>
+                    <ExtensionProvider checkoutService={checkoutService} errorLogger={errorLogger} >
+                        <MultiShippingFormV2 {...defaultProps} />
+                    </ExtensionProvider>
+                </LocaleContext.Provider>
+            </CheckoutProvider>,
+        );
+
+        await userEvent.click(screen.getByRole('button', { name: 'Add new destination' }));
```

Comment:
Shall we also get this text (`Add new destination`) from `localeContext.language`